spec/services/support_interface/ministerial_report_candidates_export_spec.rb

```diff
@@ -3,6 +3,8 @@
 RSpec.describe SupportInterface::MinisterialReportCandidatesExport do
   include StatisticsTestHelper
 
+  before { stub_const('CourseChoice', Struct.new(:name, :subjects, :status)) }
```

Comment:
Rubocop doesn't like `CourseChoice = Struct.new(...)` because it's a leaky constant.